dev.overlayed.Overlayed.yaml

```diff
@@ -0,0 +1,96 @@
+# useful info for tauri v1 https://github.com/tauri-apps/tauri/discussions/4426
+
+id: dev.overlayed.Overlayed
+
+runtime: org.gnome.Platform
+runtime-version: "46"
+sdk: org.gnome.Sdk
+
+rename-icon: Overlayed
+command: overlayed
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --socket=x11
+  - --share=ipc
+  - --share=network
+  - --device=dri
+  - --filesystem=home
+  - --socket=pulseaudio
+  - --talk-name=org.freedesktop.Notifications
+
+modules:
+  - shared-modules/libsoup/libsoup-2.4.json
+  - name: unifdef
+    no-autogen: true
+    make-install-args:
+      - prefix=${FLATPAK_DEST}
+    sources:
+      - type: archive
+        url: https://dotat.at/prog/unifdef/unifdef-2.12.tar.xz
+        sha256: 43ce0f02ecdcdc723b2475575563ddb192e988c886d368260bc0a63aee3ac400
+    cleanup:
+      - "*"
+  # NOTE: native deps
+  # https://github.com/tauri-apps/tauri-docs/blob/v1/docs/guides/building/cross-platform.md?plain=1#L113
+  # - shared-modules/libappindicator/libappindicator-gtk3-12.10.json
+  - name: webkit2gtk-4.0
+    sources:
+      - type: archive
+        url: https://webkitgtk.org/releases/webkitgtk-2.44.0.tar.xz
+        sha256: c66530e41ba59b1edba4ee89ef20b2188e273bed0497e95084729e3cfbe30c87
+        x-checker-data:
+          type: html
+          url: https://webkitgtk.org/releases/
+          version-pattern: LATEST-STABLE-(\d[\.\d]+\d)
+          url-template: https://webkitgtk.org/releases/webkitgtk-$version.tar.xz
+    buildsystem: cmake-ninja
+    config-opts:
+      # NOTE: not sure flat webkit2gtk-4.0 flags tauri requires
+      - -DPORT=GTK
+      - -DCMAKE_BUILD_TYPE=RelWithDebInfo
+      - -DENABLE_DOCUMENTATION=OFF
+      - -DENABLE_MINIBROWSER=OFF
+      - -DENABLE_WEBDRIVER=OFF
+      - -DENABLE_GAMEPAD=OFF
+      - -DUSE_LIBBACKTRACE=OFF
+      - -DUSE_GTK4=OFF
+      - -DUSE_SOUP2=ON
+      - -DENABLE_BUBBLEWRAP_SANDBOX=OFF
+  - name: overlayed
+    buildsystem: simple
+    build-commands:
+      - chmod +x Overlayed.AppImage
+      - ./Overlayed.AppImage --appimage-extract
+
+      - |
+        desktop-file-edit \
+          --set-name="Overlayed" \
+          --set-comment="Overlayed's desktop application" \
+          --set-icon="$FLATPAK_ID" \
+          --set-key="StartupNotify" --set-value="true" \
+          --set-key="Categories" --set-value="Game;" \
+          --set-key="Keywords" --set-value="discord;overlay;" \
+          --set-key="StartupWMClass" --set-value="Overlayed" \
+          squashfs-root/usr/share/applications/overlayed.desktop
```

Comment:
why do you need to change so much of the .desktop file? is it so bad in the appimage? This should be fixed upstream is that's the case.